pkg/hubble/recorder/sink/dispatch.go

```diff
@@ -78,11 +78,20 @@ type Statistics struct {
 	BytesLost      uint64
 }
 
+// StopConditions defines a set of values which cause the sink to stop
+// recording if any of them are hit. Zero-valued conditions are ignored.
```

Comment:
`Zero-valued conditions are ignored.`

do we want to pointer up all the members? that would solve the zero via nils

Reply:
What would be the use-case to support explicit zero values? It would be a stop condition that would stop the recording immediately after starting.

The technical reason why this is the way it is, is because this is populated by protobuf, which has the "absent value implies zero value" semantics. If we want to support explicit zeros (which I'm partial to, as I don't see the use-case), we would have to use the proto `Int64Value` wrappers as well.

Reply:
If we don't need to tell the difference between value,nil,zero then it's not a big deal. I suppose thinking more about it, zero stop conditions and nil conditions are the same thing..